timemachine/cpp/src/kernels/k_neighborlist.cuh

```diff
@@ -80,7 +85,7 @@ void __global__ k_find_block_bounds(
 }
 
 void __global__ k_compact_trim_atoms(
-    const int NC,
+    const int N,
```

Comment:
This seems to make the over all agorithm `O(N)` as opposed to `O(num_row_idxs)`?